tailwind/ui/__stories__/Button.stories.tsx

```diff
@@ -118,7 +117,7 @@ export const OverrideStyles: Story = {
           <MdNightlight />
         </Button>
         <ButtonLink
-          className="rounded-full px-0 py-0"
+          className="min-h-0 rounded-full px-0 py-0"
```

Comment:
@pettinarip I wonder if we can do away with the "Override Styles" story. It's now very clear that any incoming utility class from a parent component should override what is existing in the child component.

Reply:
Yeah, definitely.

Reply:
Maybe replace it with a simple ButtonLink story? I think that component is an important one and we use it a lot. I would prefer to have at least 1 story for it. What do you think?

Reply:
That I will do!